azure-communication-ui/chat/build.gradle

```diff
@@ -37,6 +37,10 @@ dependencies {
     implementation "androidx.fragment:fragment-ktx:$androidx_fragment_ktx_version"
     implementation "androidx.navigation:navigation-fragment-ktx:$androidx_navigation_fragment_ktx_version"
 
+
+    api ("com.azure.android:azure-communication-chat:1.1.0")
```

Comment:
If we decided to add dependency on azure-communication-common directly, we can make 
api for azure-communication-common
implementation for azure-communication-chat

We do not intend Contoso to use any types defined in azure-communication-chat, only once defined in common

Also please move version numbers to version.gradle on the root level as it is done in the private repo

Reply:
Will clean up the gradle after https://github.com/Azure/communication-ui-library-android/pull/415 get merged.